server/app/views/v1/containers/_container.json.jbuilder

```diff
@@ -41,3 +41,7 @@ else
   json.overlay_cidr nil
 end
 json.network_settings container.network_settings
+if container.health_status
+  json.health_status container.health_status
+  json.health_status_at container.health_status_at
```

Comment:
I've been thinking of having it as a model in a way that also can have some level of history available. This would also means that it would be easy to have more details on the health check, such as return codes, returned data etc.. That's maybe a separate PR.